core/spec/models/spree/promotion_rule_spec.rb

```diff
@@ -12,7 +12,7 @@ def eligible?
     end
 
     it "should force developer to implement eligible? method" do
-      expect { BadTestRule.new.eligible? }.to raise_error
+      expect { BadTestRule.new.eligible? }.to raise_error ArgumentError
```

Comment:
is `ArgumentError` the right error here? Sound like it should be `NotImplementedError`